sourcecode/apis/contentauthor/resources/views/games/millionaire/show.blade.php

```diff
@@ -20,6 +20,16 @@
     <script>
         var gameObject = {!!$gameSettings!!};
         var language = '{{$language}}';
+
+        // The 'hello' message is in the millionaire.js
+        window.addEventListener('message', e => {
+            if (window.parent && window.location !== window.parent.location && e.data?.action === 'hello') {
```

Comment:
Why can't the message just be sent once, preemptively? There's no point in waiting for a handshake, and it's not even a documented part of E3's resizing mechanism.

Reply:
Never mind, I understand why this is done.